tests/test_ci_insights.py

```diff
@@ -115,9 +115,7 @@ def test_qux():
 Detected 3 new tests
   - test_flaky_detection_detects_new_tests\.py::test_bar \(\d+ms\)
   - test_flaky_detection_detects_new_tests\.py::test_baz \(\d+ms\)
-  - test_flaky_detection_detects_new_tests\.py::test_qux \(\d+ms\)
-Detected 1 new flaky tests
```

Comment:
This is a very suspicious change. test_flaky_detection_detects_new_tests is supposed to detect flaky :)

Reply:
The flaky detection logic is not in this plugin now. We'll just send a span for each test and let the backend flag a test as flaky. I removed the previous code supposed to detect a flaky test and also removed this part from the report 👍